src/shared/timepicker-dropdown/timepicker-dropdown.tsx

```diff
@@ -399,6 +394,7 @@ export const TimepickerDropdown = ({
             <Container
                 ref={resizeDetector.ref}
                 data-testid={getTestId("timepicker-dropdown")}
+                inert={show ? undefined : ""}
```

Comment:
when set, the `inert` attribute disables interactivity with the element and its children. this includes exclusion from the tab sequence

it's supported in the majority of browsers now but unfortunately [still not included in React](https://github.com/facebook/react/pull/24730), so a workaround is to augment the types and whitelist it from eslint